extensions/notebook/src/common/utils.ts

```diff
@@ -121,8 +121,8 @@ export interface IEndpoint {
 	protocol: string;
 }
 
-export function getOSPlatform(): Platform {
-	switch (process.platform) {
+export function getOSPlatform(platform?: NodeJS.Platform): Platform {
```

Comment:
Honestly I'd just remove this method completely. This is just adding an extra layer for no real benefit - the return NodeJS.Platform is already a set of known values so adding our own mapping on top of it seems a bit unnecessary.

Reply:
That's fine. Let me do that. We have to keep getOSPlatformId() for now (it's how our python packages are named) but lemme rewrite some code to remove util method.

Reply:
Yup, that part is fine. It'd just switch on process.platform